packages/ibm-products/src/components/Datagrid/Datagrid/addons/Filtering/hooks/useFilters.js

```diff
@@ -334,7 +334,9 @@ const useFilters = ({
   const cancel = () => {
     // Reverting to previous filters only applies when using batch actions
     if (updateMethod === BATCH) {
-      revertToPreviousFilters();
+      if (variation !== PANEL) {
+        revertToPreviousFilters();
```

Comment:
This code introduced a bug, now when you cancel in PANEL it doesn't work 

https://github.com/carbon-design-system/ibm-products/assets/12755042/6e573bf7-c32c-4d3b-b26a-c6c084b034ad

Reply:
I believe this is resolved now with the latest changes